Formula/rm-improved.rb

```diff
@@ -26,8 +26,18 @@ def install
   end
 
   test do
-    touch "test_file"
-    system "#{bin}/rip", "--graveyard", ".graveyard", "test_file"
-    assert_predicate testpath/".graveyard", :exist?
+    trash = testpath/"trash"
+    ENV["GRAVEYARD"] = trash
+
+    source_file = testpath/"testfile"
+    deleted_file = Pathname.new File.join(trash, source_file)
```

Comment:
I think, this should work as well:
```suggestion
    deleted_file = trash / source_file
```

Reply:
I tried that but it doesn't work because both paths belong to the same ancestor lineage. I've replicated it quickly using Repl.it so you can take a look: https://repl.it/@vladimyr1/homebrew-playground

Reply:
You're right, thanks for the explanation!